src/core/core/coredump.posix.cpp

```diff
@@ -39,6 +39,7 @@
 #include <dsn/tool_api.h>
 #include <sys/types.h>
 #include <signal.h>
+#include <dsn/c/app_model.h>
```

Comment:
Why include this file?

Reply:
When I removed the include file of <dsn/service_api_c.h> in include/dsn/tool-api/logging_provider.h, the complier tells me that it can't find `dsn_exit` in `coredump.posix.cpp`